bicycleparameters/cycle_app.py

```diff
@@ -197,6 +198,13 @@ def populate_wheel_data(value, n_clicks):
         genLabels.append({'label': i})
     for i in range(8, 12):
         con.append({'con': genPure.get(pList[i])})
+
+    # converts radians to degrees for display in the table
+    lamD = con[2]
+    radians = lamD.get('con')
+    degrees = (radians*180)/math.pi
+    con[2] = {'con': int(degrees)}
```

Comment:
NumPy has the rad2deg and deg2rad functions that you may want to use in the future.